backend/pkg/api/data_access/vdb_summary.go

```diff
@@ -521,17 +521,93 @@ func (d *DataAccessService) GetValidatorDashboardGroupSummary(ctx context.Contex
 		return nil, err
 	}
 
-	getLastScheduledBlockAndSyncDate := func() (time.Time, time.Time, error) {
+	getMissedELRewards := func(epochStart, epochEnd uint64) (float64, error) {
+		// Initialize the result variable
+		var totalMissedRewardsEl float64
+
+		// Define the `targets` CTE
+		targets := goqu.
+			From("blocks").
+			Select(goqu.I("blocks.slot").As("slot")).
+			Where(
+				goqu.I("blocks.status").Neq("1"),
+				goqu.I("epoch").Gte(epochStart),
+				goqu.I("epoch").Lte(epochEnd),
+			)
+
+		if dashboardId.Validators == nil {
+			targets = targets.
+				Join(
+					goqu.T("users_val_dashboards_validators").As("uvdv"),
+					goqu.On(goqu.I("blocks.proposer").Eq(goqu.I("uvdv.validator_index"))),
+				).
+				Where(
+					goqu.And(
+						goqu.I("uvdv.dashboard_id").Eq(dashboardId.Id),
+						goqu.Or(
+							goqu.I("uvdv.group_id").Eq(groupId),
+							goqu.L("?::smallint = -1", groupId),
+						),
+					),
+				)
+		} else {
+			targets = targets.
+				Where(
+					goqu.I("blocks.proposer").In(dashboardId.Validators),
+				)
+		}
+
+		// Define the `res` CTE
+		res := goqu.
+			From("targets").
+			LeftJoin(
+				goqu.T("execution_rewards_finalized").As("b"),
+				goqu.On(
+					goqu.I("b.slot").Gte(goqu.L("targets.slot - 16")),
+					goqu.I("b.slot").Lt(goqu.L("targets.slot + 16")),
+				),
+			).
+			Select(
+				goqu.I("targets.slot"),
+				goqu.L("percentile_cont(0.5) WITHIN GROUP (ORDER BY b.value)::numeric(76,0)").As("v"),
+			).
+			GroupBy(goqu.I("targets.slot"))
+
+		// Build the final query
+		query := goqu.From("res").
+			With("targets", targets).
+			With("res", res).
+			Select(goqu.L("COALESCE(SUM(v), 0)"))
+
+		// Generate SQL and arguments
+		sql, args, err := query.ToSQL()
```

Comment:
```suggestion
		sql, args, err := query.Prepared(true).ToSQL()
```
please do this instead so goqu doesn't try to interpolate args into the query string and instead puts them into the args array. https://doug-martin.github.io/goqu/docs/interpolation.html